flask_project/campaign_manager/templates/campaign_detail.html

```diff
@@ -193,6 +207,28 @@ <h3 class="detail-campaign-name">{{ name }}</h3>
             })
         }
 
+        function findAttribution(map_url){
```

Comment:
hi @ann26 
i think this list should be done on backend
create list of map and it's attribution
and return to the frontend
custom_map = { url: <url>, attribution:<attribution> }